cmd/run/run.go

```diff
@@ -611,6 +616,7 @@ func (s *ServerContext) Run(ctx context.Context, config *serverconfig.Config) er
 
 	svr := server.MustNewServerWithOpts(
 		server.WithDatastore(datastore),
+		server.WithTokenSerializer(continuationTokenSerializer),
```

Comment:
should this be a little more specific?

```suggestion
		server.WithContinuationTokenSerializer(continuationTokenSerializer),
```